receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -312,18 +313,77 @@ func getAge(start time.Time, end time.Time) int64 {
 func (ghs *githubScraper) getCVEs(
 	ctx context.Context,
 	client graphql.Client,
+	rClient *github.Client,
 	repo string,
 ) (map[metadata.AttributeCveSeverity]int64, error) {
-	c, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo)
-	if err != nil {
-		return nil, err
+	d := ghs.getDbotAlerts(ctx, client, repo)
+	c := ghs.getCodeScanAlerts(ctx, rClient, repo)
+
+	return mapSeverities(d, c), nil
+}
+
+func (ghs *githubScraper) getDbotAlerts(
```

Comment:
```suggestion
func (ghs *githubScraper) getDepBotAlerts(
```

Reply:
`Dbot` just reads weird to me. Suggest, despite of my general preference  for brevity, I think the extra 2 letters and camel case makes sense from a readability perspective.

Reply:
Agreed, updated Dbot to DepBot for readabilty and ease of understanding. Marking as resolved